app/views/admin/orders/index.html.erb

```diff
@@ -0,0 +1,25 @@
+
+<% if flash[:errors] %>
+  <% flash[:errors].each do |error| %>
+    <p style = "color:red"><%= error %></p>
+  <% end %>
+<% end %>
+<h1>Product with Orders</h1>
+<center>
+  <table>
+    <th>Title</th>
+    <th>Quantity</th>
+    <th>Total Price</th>
+      <% @orders.each do |order| %>
```

Comment:
Use partial to display a list of orders instead of loop.

Reply:
okk